docs/src/navigation/navigationTabs/navigationTabsSampleSelectedTabKey.jsx

```diff
@@ -0,0 +1,32 @@
+import React, {
+    useState,
+} from 'react';
+import {
+    Input,
+    NavigationTabs,
+} from 'react-cm-ui';
+import { items } from '../../../../src/navigation/navigationTabs/__test__/navigationTabsMockUps';
```

Comment:
Would it be good to include the whole items array in here? (seems to be too long for the snippet but let me know), thanks.